### PR TITLE
X11-misc/xfractint: vbump, -fno-common fix

### PR DESCRIPTION
**Pull request by janko-j1**

### CODE REVIEW

File: x11-misc/xfractint/xfractint-20.04_p15.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+PATCHES=(
+	"${FILESDIR}/xfractint-20.04p09-ldflags.patch"
+)
+
+src_compile() {
+	# Untested, any x86 archteam dev. is allowed to uncomment this.
+	local myasm="foo"
+	# use x86 && myasm="/usr/bin/nasm"
+	emake CC="$(tc-getCC)" AS="${myasm}" OPT="${CFLAGS}" LDFLAGS="${LDFLAGS}"
```
Comment (juippis) [inline x11-misc/xfractint/xfractint-20.04_p15.ebuild]: This looks like some debug lines. You should probably remove them. You can use *toolchain-funcs.eclass* to get proper AS value, could be used for CFLAGS and LDFLAGS too.

**Comment by janko-j1** [comment]
Regarding https://bugs.gentoo.org/706522 and the vbump: should I kill the drop old commit and reference that bug in the vbump commit so that there is still a stable package in the tree?

File: x11-misc/xfractint/files/xfractint-20.04p15-call-gcc-directly.patch [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Fixes definition of CC variable
+
```
Comment (thesamesam) [inline x11-misc/xfractint/files/xfractint-20.04p15-call-gcc-directly.patch:2]: Could probably drop newlines between the text here

File: x11-misc/xfractint/files/xfractint-20.04p15-musl.patch [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Fix to build with musl profile
+
```
Comment (thesamesam) [inline x11-misc/xfractint/files/xfractint-20.04p15-musl.patch:2]: Any of these upstreamable btw?

File: x11-misc/xfractint/xfractint-20.04_p15.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+DEPEND="
+	${RDEPEND}
+	x11-libs/libXft
+"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+	"${FILESDIR}/xfractint-20.04p15-ldflags.patch"
```
Comment (thesamesam) [inline x11-misc/xfractint/xfractint-20.04_p15.ebuild:28]: ${PN}-20.04p15-...

File: x11-misc/xfractint/files/xfractint-20.04p15-musl.patch (hunk 3 above)
Comment (janko-j1) [reply x11-misc/xfractint/files/xfractint-20.04p15-musl.patch:2]: I'm asking right now :)

File: x11-misc/xfractint/xfractint-20.04_p16.ebuild [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+DESCRIPTION="A fractal generator"
+HOMEPAGE="https://www.fractint.org"
+SRC_URI="https://www.fractint.org/ftp/current/linux/${MY_P}.tar.gz"
+
+LICENSE="free-noncomm HPND public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="x11-libs/libX11"
```
Comment (thesamesam) [inline x11-misc/xfractint/xfractint-20.04_p16.ebuild:18]:
```
$ ~/rdeps.sh /usr/bin/xfractint
sys-libs/glibc:2.2: /lib64/libc-2.31.so
sys-libs/glibc:2.2: /lib64/libm-2.31.so
x11-libs/libX11:0: /usr/lib64/libX11.so.6.3.0
x11-libs/libXft:0: /usr/lib64/libXft.so.2.3.3
```

So, we need `libXft` at runtime too.

**Pull request closed without merging**